examples/clojure/extractJarVersion

```diff
@@ -0,0 +1,3 @@
+# Outputs the clojure project version from a passed in .jar file
+version=$(unzip -p $1 META-INF/MANIFEST.MF | grep Leiningen-Project-Version: | sed 's/^.*: //' | tr -d '\r')
```

Comment:
it would be good to test version was correctly extracted. perhaps:

```
test -n "$version" || (echo >&2 "failed to extract version" && exit 1)
```